docs/standard/serialization/xml-serializer-generator-tool-sgen-exe.md

```diff
@@ -16,19 +16,19 @@ sgen [options]
   
 |Option|Description|  
 |------------|-----------------|  
-|**/a**[**ssembly**]**:**_filename_|Generates serialization code for all the types contained in the assembly or executable specified by *filename*. Only one file name can be provided. If this argument is repeated, the last file name is used.|  
-|**/c[ompiler]:** *options*|Specifies the options to pass to the C# compiler. All csc.exe options are supported as they are passed to the compiler. This can be used to specify that the assembly should be signed and to specify the key file.|  
-|**/d**[**ebug**]|Generates an image that can be used with a debugger.|  
-|**/f[orce]**|Forces the overwriting of an existing assembly of the same name. The default is **false**.|  
+|**/a\[ssembly\]:**_filename_|Generates serialization code for all the types contained in the assembly or executable specified by *filename*. Only one file name can be provided. If this argument is repeated, the last file name is used.|  
+|**/c\[ompiler\]:** *options*|Specifies the options to pass to the C# compiler. All csc.exe options are supported as they are passed to the compiler. This can be used to specify that the assembly should be signed and to specify the key file.|  
```

Comment:
Not related to this PR per-se, but is the space here invalid? It's at least inconsistent with other rows.

```suggestion
|**/c\[ompiler\]:**_options_|Specifies the options to pass to the C# compiler. All csc.exe options are supported as they are passed to the compiler. This can be used to specify that the assembly should be signed and to specify the key file.|
```